crypto/src/main/scala/org/bitcoins/crypto/AdaptorUtil.scala

```diff
@@ -28,13 +46,13 @@ object AdaptorUtil {
   def adaptorSign(
```

Comment:
So if I understand correctly, we take in `dataToSign` (called `message` in the spec, i would be in favor of renaming our parameter name to this) so we can preserve this invariant right? 

>message_hash: A 32-byte array which must be the hash of the message (in our case a transaction digest).

If so, it might be worth noting the scaladoc why we take the message and not just the hash. 

As a side note, it might be worth adding to the spec itself _why_  this MUST be the hash of the message. I don't understand if it will just fail to verify, or we can do something more sinister like leak a private key.